tfjs-backend-webgpu/src/shader_preprocessor_wgsl.ts

```diff
@@ -226,8 +226,17 @@ const SHADER_PREFIX = `
   }
 
   fn greaterThanVec4F32(a : vec4<f32>, b : vec4<f32>) -> vec4<f32> {
-    let resultBool = vec4<bool>(a > b);
-    return vec4BoolToVec4F32(resultBool);
+    var res = vec4<f32>(0.0);
```

Comment:
Please add a comment to point to the bug you reported and state that it is a workaround. Similar for other places.

Reply:
Related workaround removed, PTAL.